docs/API/fliplet-communicate.md

```diff
@@ -60,6 +58,18 @@ Fliplet.Communicate.sendSMS(options);
 
 Let us know if you require to use another SMS provider and we'll check whether we can integrate it on our system.
 
+## Compose an email
+
+```js 
+// TBC
+```
+
+## Share a URL
+
+```js 
+Fliplet.Communicate.shareURL('https://maps.google.com/?addr=N1+9PF');
```

Comment:
```js
Fliplet.Communicate.share({ url: 'http://' })
```

because Android and iOS both supports sharing different types of content (e.g. text, image and so on)

Reply:
I actually intend on doing exactly what you have suggested, but we only need support for URL at the moment (and I only want to test for it). So, the intention is to have a `.shareUrl()` that maps to `Fliplet.Communicate.share({ url })`.

Reply:
@tonytlwu that works. Another way is also to allow `share('http://')` as a string. In the future we can check whether it's a URL when we add more types to be shared.

Reply:
@squallstar I think we're synchronised on how we think now :D I also had that in mind, though I think looking at the rest of the `Fliplet.Communicate` functions, I decided being specific like `.shareURL()` would be better because we didn't go for `.email({ type: 'send' })` vs `.email({ type: 'compose' })`. We have been very specific with the `Fliplet.Communicate` function names.